packages/asc-ui/src/components/CardContainer/CardContainer.stories.tsx

```diff
@@ -0,0 +1,438 @@
+  Api,
+  Data,
+  Table,
+  Pano,
+  Map,
+} from '@datapunt/asc-assets'
+import { storiesOf } from '@storybook/react'
+import styled from '@datapunt/asc-core'
+import React from 'react'
+import { color, breakpoint } from '../../utils'
+import Icon from '../Icon'
+import IconStyle from '../Icon/IconStyle'
+import CardContainer from '.'
+import Card from '../Card/Card'
+import CardStyle from '../Card/CardStyle'
+import CardContentStyle from '../Card/CardContentStyle'
+import { CardMediaWrapperStyle } from '../Card/CardMediaStyle'
+import CardActions from '../Card/CardActions'
+import CardActionsStyle from '../Card/CardActionsStyle'
+import CardContent from '../Card/CardContent'
+import CardMedia from '../Card/CardMedia'
+import Heading from '../Heading'
+import HeadingStyle from '../Heading/HeadingStyle'
+import Paragraph from '../Paragraph'
+import ParagraphStyle from '../Paragraph/ParagraphStyle'
+import Row from '../Grid/Row'
+import Column from '../Grid/Column'
+import ColumnStyle from '../Grid/ColumnStyle'
+import Link from '../Link'
+import LinkStyle from '../Link/LinkStyle'
+
+const CardContainerGenericFunctions = styled(CardContainer)`
+  position: relative;
+  padding: 12px;
+  background-color: ${color('support', 'valid')};
+
+  @media screen and ${breakpoint('min-width', 'mobileM')} {
+    padding: 24px;
+  }
+
+  // Separate content in two columns on tabletM only
+  @media screen
+  and ${breakpoint('min-width', 'tabletM')}
+  and ${breakpoint('max-width', 'laptop')} {
+    column-count: 2;
+    column-gap: 8px;
+
+    // Fallback in case of having an unequal count of cards
+    // eg: 5 cards instead of 6
+    // because that breaks the design in Firefox and Safari
+    padding-bottom: calc(24px - 8px); // Calculate the padding-bottom minus margin-bottom of the card
+    ${CardStyle} {
+      display: inline-flex;
+      width: 100%;
+      break-inside: avoid;
+    }
+  }
+
+  // Generate the arrow / triangle in the left corner
+  @media screen and ${breakpoint('min-width', 'tabletM')} {
+    ::after {
+      content: '';
+      position: absolute;
+      height: 0;
+      width: 0;
+      left: 0px;
+      top: 0px;
+      border-style: solid;
+      border-width: 24px 24px 0 0;
+      border-color: #fff transparent transparent transparent;
+    }
+  }
+
+  ${LinkStyle} {
+    position: relative;
+    width: 100%;
+    height: 73px;
+    margin-bottom: 8px;
+
+    &:hover ${HeadingStyle} {
+      color: ${color('secondary')};
+      text-decoration: underline;
+    }
+
+    &:focus {
+      background: none;
+
+      ${CardStyle}:after {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        bottom: 0;
+        right: 0;
+
+        ${({ theme }) =>
+          `border: 4px solid ${color('support', 'focus')({ theme })};`}
+      }
+    }
+  }
+
+  ${CardStyle} {
+    // height: 73px;
+  }
+
+  ${CardMediaWrapperStyle} {
+    @media screen and ${breakpoint('max-width', 'mobileL')} {
```

Comment:
see previous comment, if we use percentage we might not have this problem (but not sure though)